packages/blade/src/components/SkipNav/__tests__/SkipNav.native.test.tsx

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import renderWithTheme from '../../../_helpers/testing/renderWithTheme.web';
+import { SkipNavContent, SkipNavLink } from '../SkipNav';
+
+describe('<SkipNav />', () => {
+  it('<SkipNavLink /> should throw error on native', () => {
+    expect(() => renderWithTheme(<SkipNavLink />)).toThrow();
+  });
+
+  it('<SkipNavContent /> should throw error on native', () => {
+    expect(() => renderWithTheme(<SkipNavContent />)).toThrow();
```

Comment:
sir, make it verbose 

[Ref](https://github.com/razorpay/blade/blob/master/packages/blade/src/components/Typography/BaseText/__tests__/BaseText.native.test.tsx#L52-L75)

Reply:
Done